src/library_idbfs.js

```diff
@@ -280,6 +280,12 @@ addToLibrary({
         }
       };
 
+      // transaction may abort if (for example) there is a QuotaExceededError
+      transaction.onabort = (e) => {
+        done(event.target.error); // DOMException
```

Comment:
Whats the difference between using `e` and `e.event.target` here and `error.event.target` here?

Reply:
Great catch - should be `e`. Fixed.